packages/components/radio/src/radio.js

```diff
@@ -0,0 +1,63 @@
+import { TSElement, unsafeCSS, html, customElementDefineHelper } from '@tradeshift/elements';
+import css from './radio.css';
+import { customEventNames } from './utils/constants';
+
+customElementDefineHelper(
+	'ts-radio',
+	class extends TSElement {
+		static get styles() {
+			return [TSElement.styles, unsafeCSS(css)];
+		}
+
+		static get properties() {
+			return {
+				value: { type: String, reflect: true },
+				label: { type: String, reflect: true },
+				checked: { type: Boolean, reflect: true },
+				disabled: { type: Boolean, reflect: true },
+				dir: { type: String, reflect: true }
```

Comment:
Why did you add the dir to radio itself? I think only radio-group should have the dir.

Reply:
Yes, it's much more logical to set dir only on the parent. The current implementation hasn't changed from the first version, where each radio was an independent entity.

Reply:
Fixed